platform/pulpcore/tasking/celery_app.py

```diff
@@ -91,18 +91,20 @@ def stop(self, consumer):
             self.timer_ref.cancel()
             self.timer_ref = None
 
-    def _record_heartbeat(self, consumer):
+    def _handle_heartbeat(self, consumer):
```

Comment:
Perhaps this should be renamed to ```on_tick()``` or something more generic.  Ditto to the name of the class -- maybe name it something like ```PulpWorkerStep```

The docstring here should also be improved -- maybe "This method performs cleanup for dead workers, and records a heartbeat for the current worker"